tests/unit/test_req_install.py

```diff
@@ -42,3 +45,50 @@ def test_forward_slash_results_in_a_link(self, tmpdir):
         )
 
         assert requirement.link is not None
+
+
+class TestInstallRequirementFrom(object):
+
+    def test_install_req_from_string_without_comes_from(self):
+        """
+        Test to make sure that install_req_from_string succeeds
+        when called with URL (PEP 508) but without comes_from.
+        """
+        # Test with a PEP 508 url install string:
+        wheel_url = ("https://download.pytorch.org/whl/cu90/" +
```

Comment:
You're going to get a style error here, but note that you don't need a `+` (parentheses concatenate without that).

Reply:
You might need to do something like the following to address linter complaints--

```python
wheel_url = (
    "https://download.pytorch.org/whl/cu90/..."
)
```

Reply:
Now done?